internal/util/bitvect.go

```diff
@@ -0,0 +1,655 @@
+package util
+
+import (
+	"fmt"
+	"sync"
+)
+
+// A BitVect is a matrix of 512 by 512 bits encoded into uint64 elements.
+type BitVect struct {
+	set [512 * 8]uint64
```

Comment:
It should be `(512/64) * (512/64) = [64]uint64`. That's going to trickle in all files.

Reply:
My bad, this is the right thing.